msvc-full-features/ImGui-lib-vcpkg-static.vcxproj

```diff
@@ -0,0 +1,192 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project DefaultTargets="Build" ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <ItemGroup Label="ProjectConfigurations">
+    <ProjectConfiguration Include="Debug|Win32">
+      <Configuration>Debug</Configuration>
+      <Platform>Win32</Platform>
+    </ProjectConfiguration>
+    <ProjectConfiguration Include="Debug|x64">
+      <Configuration>Debug</Configuration>
+      <Platform>x64</Platform>
+    </ProjectConfiguration>
+    <ProjectConfiguration Include="Release|Win32">
+      <Configuration>Release</Configuration>
+      <Platform>Win32</Platform>
+    </ProjectConfiguration>
+    <ProjectConfiguration Include="Release|x64">
+      <Configuration>Release</Configuration>
+      <Platform>x64</Platform>
+    </ProjectConfiguration>
+  </ItemGroup>
+  <ItemGroup>
+    <ClInclude Include="..\src\third-party\imgui\imconfig.h" />
+    <ClInclude Include="..\src\third-party\imgui\imgui.h" />
+    <ClInclude Include="..\src\third-party\imgui\imgui_impl_sdl2.h" />
+    <ClInclude Include="..\src\third-party\imgui\imgui_impl_sdlrenderer.h" />
+    <ClInclude Include="..\src\third-party\imgui\imgui_internal.h" />
+    <ClInclude Include="..\src\third-party\imgui\imstb_rectpack.h" />
+    <ClInclude Include="..\src\third-party\imgui\imstb_textedit.h" />
+    <ClInclude Include="..\src\third-party\imgui\imstb_truetype.h" />
+  </ItemGroup>
+  <ItemGroup>
+    <ClCompile Include="..\src\third-party\imgui\imgui.cpp" />
+    <ClCompile Include="..\src\third-party\imgui\imgui_demo.cpp" />
+    <ClCompile Include="..\src\third-party\imgui\imgui_draw.cpp" />
+    <ClCompile Include="..\src\third-party\imgui\imgui_impl_sdl2.cpp" />
+    <ClCompile Include="..\src\third-party\imgui\imgui_impl_sdlrenderer.cpp" />
+    <ClCompile Include="..\src\third-party\imgui\imgui_tables.cpp" />
+    <ClCompile Include="..\src\third-party\imgui\imgui_widgets.cpp" />
+  </ItemGroup>
+  <PropertyGroup Label="Globals">
+    <ProjectGuid>{0C0B2BEA-311F-473C-9652-87923EF639E3}</ProjectGuid>

```

Comment:
Better yet use vcpkg port of imgui (possibly with local patches).

Reply:
this should be resolved, I am pretty confident that I rebuilt the project file as needed... BTW if we are renaming the Imgui project then I'm not sure why we don't rename most of the rest of them, since the naming convention seeems standard in CDDA.